PingClient/Program.cs

```diff
@@ -1 +1,34 @@
-﻿await PingClient.Client.ConnectAndDisconnectClientUsingMQTTv5();
+﻿

```

Comment:
Pack all the code below to public member function Client::run() so there is encapsulation in Client class. Then you can simply run client.run()